tpu-client-next/src/connection_workers_scheduler.rs

```diff
@@ -103,7 +108,7 @@ impl ConnectionWorkersScheduler {
     pub async fn run(
```

Comment:
might be nice to include a comment about why we're returning the receiver (like what you have in the PR description)

Reply:
yeah, I forgot to add it to the comment.